# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ecore/legacy_sus.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

usleep() was deprecated by SUSv3 and newer LIBCs are starting to enforce that
deprecation.

--- ./src/lib/ecore_fb/ecore_fb_vt.c.orig	2010-09-20 17:17:59.000000000 -0400
+++ ./src/lib/ecore_fb/ecore_fb_vt.c	2010-09-20 17:21:28.000000000 -0400
@@ -127,7 +127,7 @@
 	/* What does this does? */
 	_ecore_fb_filter_handler = ecore_event_filter_add(_ecore_fb_event_filter_start, _ecore_fb_event_filter_filter, _ecore_fb_event_filter_end, NULL);
 	
-	usleep(40000);
+	nanosleep(&((struct timespec) {0, 40000000L}), NULL);
 	if(ioctl(_ecore_fb_vt_tty_fd, VT_ACTIVATE, _ecore_fb_vt_current_vt) < 0)
 	{
 		perror("[ecore_fb:vt_setup] error on VT_ACTIVATE");
